news/2022-10-03-results-triangles.md

```diff
@@ -0,0 +1,153 @@
+
+Major props to everyone involved in this future classic!
+
+## Full results
+
+If you want all the juicy judging details, you're in the right place.
+
+Here's the breakdown of scores:
+
+| Rank | Team submission | Mapper panel | Storyboarder panel | Averaged total |
+| :-- | :-- | :-- | :-- | :-- |
+| \#1 | [Crazy Thursday](https://osu.ppy.sh/beatmapsets/1841885) | 143 | 127 | **254.11** |
+| \#2 | [Crush the Candy Boys](https://osu.ppy.sh/beatmapsets/1841845) | 142 | 105 | **231.22** |
+| \#3 | [三角形.tar.gz](https://osu.ppy.sh/beatmapsets/1842041) | 84 | 149 | **223.67** |
+| \#4 | [absolutno pofig](https://osu.ppy.sh/beatmapsets/1841846) | 81 | 101 | **173** |
+| \#5 | [Δ = b² – 4ac](https://osu.ppy.sh/beatmapsets/1842004) | 78 | 84 | **153.33** |
+| \#6 | [?](https://osu.ppy.sh/beatmapsets/1841847) | 51 | 106 | **151.33** |
+| \#7 | [TV Sizers](https://osu.ppy.sh/beatmapsets/1841923) | 115 | 48 | **150.22** |
+| \#8 | [ULTRA ICE TEA](https://osu.ppy.sh/beatmapsets/1841926) | 68 | 84 | **144.44** |
+| \#9 | [Team Funny](https://osu.ppy.sh/beatmapsets/1842149) | 88 | 26 | **104.2222222** |
+| \#10 | [GokuLook Princesa](https://osu.ppy.sh/beatmapsets/1841839) | 69 | 42 | **103.33** |
+| \#11 | [Penta-force](https://osu.ppy.sh/beatmapsets/1841996) | 36 | 35 | **67** |
+| \#12 | dreamers | 9 | 8 | **16** |
+
+Judging for this contest involved some experimentation. Instead of assigning raw numbers to submission like most contests, the judges were given two tasks:
+
+1. Order the submissions from best to worst in their respective expertise (mapping or storyboarding).
+2. Assign a limited supply of bonus points based on standard judging categories.
+   - Mapping panel: technical skill, creativity, theme, outstanding Easy/Normal/Hard/Insane/Expert difficulty 
+   - Storyboarding panel: technical skill, creativity, theme, general storyboard impression
+
+The first task earned a submission between 1 and 12 points (12 points for a judge's #1 map, 11 for #2, 10 for #3, etc.). Each bonus point added one additional point to the total. See the [judging compilation sheet](https://docs.google.com/spreadsheets/d/1VwajV4Ibn-k_sNHiiPAXX0raXiX34-v7_6SBWUq2pPY/edit#gid=0) to see the full judging breakdown, including all map quality orders, bonus point distributions, and judge comments!
+
+## triangles Playlist leaders
+
+There's one more set of prizes to dish out today, but they're not for mappers or storyboarders!
+
+To vote for this contest, we required the community to **play each of the submission in osu!(lazer)** through one of the five difficulty-specific playlists (Easy/Normal/Hard/Insane/Expert).
+
+As an added bonus, we also decided to reward the players at the top of each playlist with a month of osu!supporter, plus the **triangles aficionado profile badge** for the winner of the Expert playlist!
```

Comment:
based on a [previous review on the same thing](https://github.com/ppy/osu-wiki/pull/7947#discussion_r962183926) i guess it's intended like this?

```suggestion
As an added bonus, we also decided to reward the players at the top of each playlist with a month of osu!supporter, plus the **triangle aficionado profile badge** for the winner of the Expert playlist!
```